federation/pkg/dnsprovider/providers/google/clouddns/rrset.go

```diff
@@ -30,6 +32,10 @@ type ResourceRecordSet struct {
 	rrsets *ResourceRecordSets
 }
 
+func (rrset ResourceRecordSet) String() string {
+	return fmt.Sprintf("<(clouddns) %q type=%s rrdatas=%q ttl=%v>", rrset.Name(), rrset.Type(), rrset.Rrdatas(), rrset.Ttl())
```

Comment:
I figured that in places where we would stringify this, it's obvious that it's a ResourceRecordSet, and only the implementation would be needed to disambiguate it further.  I'm really off in the weeds here making up a format on my own and I'm happy to change it.

Or maybe it would be best to file a bug saying "we need a style guide, and it should include how we stringify things".